### ISSUE TITLE
Error with  maven-source-plugin when activate explicit profiles

### ISSUE THREAD

**Issue by kthoms** [operaton/operaton#130]
**Describe the bug**
A build error occurs:
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-source-plugin:3.3.1:jar-no-fork (attach-sources) on project operaton-engine: Presumably you have configured maven-source-plugin to execute twice in your build. You have to configure a classifier for at least one of them. -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.apache.maven.plugins:maven-source-plugin:3.3.1:jar-no-fork (attach-sources) on project operaton-engine: Presumably you have configured maven-source-plugin to execute twice in your build. You have to configure a classifier for at least one of them.
```   

**To Reproduce**
Run 
```
./mvnw package -DskipTests -Pdistro,distro-wildfly,distro-webjar,javadocs
```

**Comment by shiftyseaotter** [comment]
I have not been able to reproduce it locally using your example and trying different profile combinations (I have not tried all possible combinations) and the latest build. It may have been accidentally fixed by now

**Comment by javahippie** [comment]
Thanks for picking this up, @shiftyseaotter. I tried replicating it with `./mvnw package -DskipTests -Pdistro,distro-wildfly,distro-webjar,javadocs` too, and the build was successful for me, too. 

I saw this error in the release build before it was finished, was this maybe fixed while working on the release?

**Comment by kthoms** [comment]
I need to check this again. While working on the release, I was changing also the usage of the source plugin. It is active with the new `sources` profile. So normal builds won't execute the plugin anymore.
It can be activated with listing `sources`as profile or `with-sources` as sys prob.

**Comment by kthoms** [comment]
Fixed via #91b56b148d5bbaab22936e9840290a5e07b53823 as part of #37
